src/exec.js

```diff
@@ -12,6 +12,33 @@ const exec = (command, args, stdin, cwd) => {
 
     const process = childProcess.spawn(command, args, spawnOptions)
 
+    // All of these handlers can close the Promise, so guard closing it twice.
+    let promiseIsClosed = false
+
+    process.on('error', err => {
```

Comment:
I'm not sure this `process.on('error'` is actually necessary; I can try removing it. What is necessary is the next handler, the `process.stdin.on('error`.

The error happens when the process is closed, but is actually generated by the process's stdin. The chain of events is:
1. We are still writing to stdin.
2. Child process closes for some reason (such as an invalid jq filter).
3. When the child process closes, its stdin closes as well.
4. We continue to try to write to its stdin.
5. Its stdin generates an error, because we're trying to write to it but it's been closed.